extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -19,51 +19,44 @@
 amp-social-share {
   height: 44px;
   width: 60px;
-}
-
-amp-social-share span {
-  display: block;
-  border-radius: 5px;
-  height: 100%;
-  width: 100%;
-}
-
-amp-social-share a {
-  display: block;
-  width: 100%;
-  height: 100%;
   background-repeat: no-repeat;
   background-position: center;
   background-size: contain;
   text-decoration: none;
+  cursor: pointer;
+}
+
+amp-social-share.-amp-layout-container {
+  display: inline-block;
+  vertical-align: top;
```

Comment:
For some reason, when content is set inside the `amp-social-share` the vertical alignment of the element got messed up. I can demo it to you if you'd like.